src/vaadin-text-field-mixin.html

```diff
@@ -776,11 +835,15 @@
       this[previousSizePropertyName] = value;
     }
 
+    /** @private */
     __observeOffsetHeight() {
       this._dispatchIronResizeEventIfNeeded('Height', this.offsetHeight);
     }
 
     /**
+     * @param {string} prop
+     * @param {unknown} oldVal
+     * @param {unknown} newVal
      * @protected
      */
     attributeChangedCallback(prop, oldVal, newVal) {
```

Comment:
`oldVal` and `newVal` types could also be string since these are attribute values which can only be strings (or also undefined in this case I think). Should also maybe add the missing `namespace` param that is documented on Polymer even if we don't use it.

See https://polymer-library.polymer-project.org/3.0/api/polymer-element#PolymerElement-method-attributeChangedCallback

Reply:
Done.